anomaly-event-processor/build.gradle.kts

```diff
@@ -0,0 +1,57 @@
+plugins {
+    java
+    application
+    jacoco
+    id("org.hypertrace.jacoco-report-plugin")
+    id("org.hypertrace.docker-java-application-plugin")
+    id("org.hypertrace.integration-test-plugin")
+}
+
+application {
+    mainClass.set("org.hypertrace.core.serviceframework.PlatformServiceLauncher")
+}
+
+tasks.test {
+    useJUnitPlatform()
+}
+
+dependencies {
+    implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.23")
+    implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.23")
+    implementation("org.hypertrace.core.documentstore:document-store:0.5.7")
+
+    implementation(project(":metric-anomaly-data-model"))
+    implementation(project(":notification-transport"))
+
+    implementation("com.sendgrid:sendgrid-java:4.7.1")
+    implementation("org.hypertrace.core.query.service:query-service-client:0.5.2")
```

Comment:
Seems like some of the clients is not required here. Can we remove un-used libs?